content/en/docs/demos/bidirection_tls.md

```diff
@@ -0,0 +1,937 @@
+---
```

Comment:
This is for Nginx ingress controller. We should add an integration doc with fsm too.